examples/index.json

```
@@ -992,31 +972,31 @@
         "category": "Working over the network",
         "samples": [
             {
-                "name": "Artemis ANYCAST consumer",
+                "name": "ANYCAST consumer",
                 "url": "artemis-anycast-session-consumer"
             },
             {
-                "name": "Artemis ANYCAST producer",
+                "name": "ANYCAST producer",
```

```suggestion
                "name": "ANYCAST Producer",
```
Shouldn't it be in uppercase to be consistent with the rest (e.g., "Basic Consumer" above).